lib/bin/cli.js

```diff
@@ -43,6 +43,8 @@ program.command('build')
   .action((command) => {
     // Set NODE_ENV to 'production'
     process.env.NODE_ENV = 'production'
+    process.env.PREFIX_LINKS = command.prefixLinks && 'prefix-links'
```

Comment:
Ok, got my head around this now I think. I don't think we need to set any environment variable for this right? Since you have access to the `program` variable in the webpack config so you can pass the prefix links state directly to the plugin?

Don't like creating globals if not needed.

I really like how `create-react-app` handles things. There's another issue open for this if you feel like fixing this with a new PR :-) https://github.com/gatsbyjs/gatsby/issues/660#issuecomment-280184453